scripts/us_epa/airqualityindex/air_quality_aggregate.py

```diff
@@ -101,12 +101,32 @@ def write_csv(csv_file_path, reader):
 
 
 def request_and_write_csv(csv_file_path, filename):
-    response = requests.get(
-        f'https://aqs.epa.gov/aqsweb/airdata/{filename}.zip')
-    with zipfile.ZipFile(io.BytesIO(response.content)) as zf:
-        with zf.open(f'{filename}.csv', 'r') as infile:
-            reader = csv.DictReader(io.TextIOWrapper(infile, 'utf-8'))
-            write_csv(csv_file_path, reader)
+    try:
```

Comment:
Change the condition in line 62 from
 if 'State Code' and 'County Code' in observation:
to  if 'State Code' in observation and 'County Code' in observation:
as if 'State Code' is always true even if 'State Code' is no in data.